nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/ProcessorNode.java

```diff
@@ -300,4 +301,12 @@ public abstract CompletableFuture<Void> stop(ProcessScheduler processScheduler,
      * @param context The ProcessContext associated with the Processor configuration
      */
     public abstract void onConfigurationRestored(ProcessContext context);
+
+    /**
+     * Returns those parameter references which configures the Processor through Parameters. This is for Processor configuration only,
+     * Processor Property Parameter references stored and handled separately.
+     *
+     * @return the configuration Parameter references for this Processor
+     */
+    public abstract List<ParameterReference> getConfigurationParameterReferences();
```

Comment:
Not sure that the naming convention here makes sense. Parameters are always used for configuration. Processor properties are very much configuration.

Reply:
This method has been removed.